.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,13 @@
+contact_links:
```

Comment:
1. What is this file used for?
2. Where is documentation on this templating methodology?

Reply:
## 1) It's adding links to other resources
<img width="1153" alt="image" src="https://github.com/PalisadoesFoundation/talawa-admin/assets/76090263/5670650a-2ba0-4855-8a4a-9c6d62abd679">

## 2) https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms

Reply:
You can also try to edit the yml file and it automatically gives the complete formatting on right side:
-- 
<img width="1440" alt="image" src="https://github.com/PalisadoesFoundation/talawa-admin/assets/76090263/a3664ff6-e67d-4ebf-aed2-6c0d681a925f">

Reply:
Remove all the links to creating issues in other repositories. They are not relevant